kitsune/questions/views.py

```diff
@@ -509,6 +514,7 @@ def aaq(request, product_key=None, category_key=None, step=1):
     if step == 2:
         context["featured"] = get_featured_articles(product, locale=request.LANGUAGE_CODE)
         context["topics"] = topics_for(product, parent=None)
+        context["zendesk_form"] = has_subscriptions
```

Comment:
s/zendesk_form/zendesk_form_exists or even better a name related to the status of the product. Eg produc_has_subscriptions/has_subscriptions